cmd/soroban-cli/src/commands/contract/build.rs

```diff
@@ -110,17 +112,20 @@ pub enum Error {
     WritingWasmFile(io::Error),
     #[error("invalid meta entry: {0}")]
     MetaArg(String),
+    #[error("rust version {0} cannot be used to build contracts")]
```

Comment:
Can we include the ranges of contracts supported?

Reply:
Does this look better? It's tricky to write a good message for this.

`use rust 1.81 or 1.84+ to build contracts (got {0})`